extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -396,7 +396,7 @@ export class AmpUserNotification extends AMP.BaseElement {
 
   /** @override */
   show() {
-    setStyle(this.element, 'display', '');
+    toggle(this.element, true);
```

Comment:
Does this still display the user notification? At this point, looks like it's still `display: none` and calling `toggle()` won't undo that.